components/suggest/src/rs.rs

```diff
@@ -214,6 +202,15 @@ pub(crate) enum SuggestRecord {
     Geonames,
 }
 
+impl SuggestRecord {
```

Comment:
This feels a little goofy just so that we can use `Record` to `get_attachment`, but setting `fields` to `Map::new( )` also felt weird

Reply:
Yes, slightly goofy but this makes sense to me.